~~~PATCH 114626ac313d886be0778f3034f60253:db8f138607142e9b1ba3b458974fa68b lib/Foswiki/Configure/Checker.pm (Foswiki-1.1.1,Foswiki-1.1.2)
diff --git lib/Foswiki/Configure/Checker.pm lib/Foswiki/Configure/Checker.pm
index 17bcbe6..dab4736 100644
--- lib/Foswiki/Configure/Checker.pm
+++ lib/Foswiki/Configure/Checker.pm
@@ -456,7 +456,11 @@ sub checkRCSProgram {
             $err .= $this->ERROR( $prog
                   . ' did not return a version number (or might not exist..)' );
         }
-        if ( $version =~ /^\d/ && $version < $rcsverRequired ) {
+
+        # Item11955 - '5.8.1' < 5.7 results in a warning (non-numeric compare)
+        # Best practice is to use CPAN:version, but isn't core until perl 5.10.
+        # So instead let's make the comparison work by stripping out sub-decimal
+        if ( $version =~ /\D(\d+(\.\d+)?)/ && $1 < $rcsverRequired ) {
 
             # RCS too old
             $err .=
